components/cloud-event-gateway-proxy/pkg/health/probes.go

```diff
@@ -0,0 +1,30 @@
+package health
+
+import (
+	"net/http"
+)
+
+const (
+	livenessURI  = "/healthz"
+	readinessURI = "/readyz"
+)
+
+func WithLivenessCheck(next http.Handler) http.Handler {
```

Comment:
I wonder whether we shouldn't merge liveness and ready probe checks into one function (because of code duplication). Based on the `request.RequestURI` the HTTP response could be different in the future if both probes diverge, but for now it is the same.

Reply:
Done.